models/yield_yak/arbitrum/yield_yak_arbitrum_schema.yml

```diff
@@ -94,7 +91,149 @@ models:
         description: ""
       - &evt_index
         name: evt_index
-        description: ""
+        description: "The position of this event log within this transaction"
       - &block_month
         name: block_month
         description: "UTC event block month of each DEX trade"
+  - name: yield_yak_arbitrum_deposits
+    meta:
+      blockchain: arbitrum
+      project: yield_yak
+      contributors: angus_1
+    config:
+      tags: ['arbitrum', 'yield', 'aggregator', 'yield_yak', 'auto-compound', 'deposits']
+    description: >
+        yield yak deposits
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - blockchain
+            - contract_address
+            - evt_index
+            - tx_index
+            - block_number
```

Comment:
each of these four spells unique combo of columns tests have different columns compared to the unique keys in the model config blocks. can you update these to reflect the same columns as the model? or the opposite, whichever is needed?

Reply:
same applies to avax file below

Reply:
Yep, I'll make that change, thanks! I think strictly speaking the `contract_address` does not need to appear in the unique combo as `block_number`, `tx_index` and `evt_index` should be sufficient for uniqueness so I'll remove `contract_address` from the arbitrum/avax schemas and the top-level schema, but I'll keep `blockchain` in the top-level schema that unions arbitrum and avax data.

Reply:
Actually for balances it's a little different (still need the `contract_address`) but will get them to match by removing `blockchain` from the arbitrum/avax schemas.